### ISSUE TITLE
training problem

### ISSUE THREAD

**Issue by xueyaosmile** [HighwayWu/InpaintingForensics#7]
I used the repair data set I made for training. After the first epoch, the loss and loss remained unchanged. In addition, the loss kept increasing during the training process. What is the problem?
![1](https://user-images.githubusercontent.com/27891522/148727151-60717d97-4767-4e5b-bd51-44bfb5435c9f.PNG)

**Comment by HighwayWu** [comment]
Please double-check that the training data (or the generation of npy file) is correctly loaded.

**Comment by xueyaosmile** [comment]
> 请仔细检查训练数据（或 npy 文件的生成）是否正确加载。

![image](https://user-images.githubusercontent.com/27891522/148960019-f514abb2-5706-402d-bb35-c03f4cb3e552.png)
Hello, I checked the data format, is this the case, or the problem still occurs

**Comment by xueyaosmile** [comment]
> 请仔细检查训练数据（或 npy 文件的生成）是否正确加载。

![image](https://user-images.githubusercontent.com/27891522/148967256-6a11d304-23e9-4611-a04d-43996d5e021d.png)
Hello, I checked the data format, as shown below, there should be no problem, but the loss still increases during training, and the acc becomes 50

**Comment by HighwayWu** [comment]
Could you please try the training by using our GC_Places/Dresden data? And if it works, the problem may be your dataset is not suitable for this Xception-based network.

**Comment by xueyaosmile** [comment]
> Could you please try the training by using our GC_Places/Dresden data? And if it works, the problem may be your dataset is not suitable for this Xception-based network.

Hello, I am training on the places dataset given by the author.
